awesome_owl/static/src/playground.js

```diff
@@ -1,7 +1,19 @@
 /** @odoo-module **/
 
-import { Component } from "@odoo/owl";
-
+import { Component, markup, useState } from "@odoo/owl";
+import { Counter } from "./counter/counter.js"
+import { Card } from "./card/card.js"
+import { TodoList } from "./todo/todoList.js";
 export class Playground extends Component {
-    static template = "awesome_owl.playground";
+    static template = "awesome_owl.Playground";
+    static components = { Counter, Card, TodoList }
+
+    setup() {
+        this.state = useState({ counter_sum: 2 })
```

Comment:
We use camel case and don't forget the semi-colon at the end...
```suggestion
        this.state = useState({ counterSum: 2 });
```